__tests__/shortsha.test.ts

```diff
@@ -0,0 +1,10 @@
+import {shortsha} from '../src/slug'
+
+function test_short_sha(input: string, expected: string) {
```

Comment:
https://github.com/rlespinasse/github-slug-action/pull/21#discussion_r482905369 from @ChristianUlbrich

Reply:
I let you choose what you prefer @rlespinasse.